index.html

```diff
@@ -11,91 +11,54 @@
-                <div class="card_topic mb-1">White Sneakers</div>
-                <div class="card_author">from ₹1999/- only</div>
-            </div>
-            <div class="card_content pl-3 pr-1 mt-2">FAUSTO Men's Lace Up Sneakers for Casual Outfit</div>
-            <div class="card_footer mt-2 mb-2 pl-3">
-                <div class="card_buttons">
-                    <div class="card_button p-1"><i class="fa fa-shopping-cart mr-1"></i> Add to cart</div>
-                    <div class="card_button p-1">Buy Now</div>
+    <p class="heading1 center">Product Categories</p>
+    <div class="category">   
+            <div class="card m-2 up-curve-border">
+                <img class="up-curve-border" src='./assets/casual-sportss.jpg' />
+                <div class="card_main pl-3 mt-2 mb-1">
+                    <div class="card_topic mb-1">Casual Shoes</div>
                 </div>
-            </div>
-        </div>
-        
-        <div class="card m-2 up-curve-border">
-            <img class="card-img up-curve-border" src="./assets/casual-sportss.jpg"></img>
-            <div class="card_main pl-3 mt-2 mb-1">
-                <div class="card_topic mb-1">Casual Sports Shoes</div>
-                <div class="card_author">from ₹1499/- only </div>
-            </div>
-            <div class="card_content pl-3 pr-1 mt-2">Textile Low Boot Laces Unisex Sports Shoes</div>
-            <div class="card_footer mt-2 mb-2 pl-3">
-                <div class="card_buttons">
-                    <div class="card_button p-1"><i class="fa fa-shopping-cart mr-1"></i> Add to cart</div>
-                    <div class="card_button p-1">Buy Now</div>
+                <div class="card_content pl-3 pr-1 mt-2">Textile Low Boot Laces Unisex Sports Shoes</div>
+                <div class="card_footer mt-2 mb-2 pl-3">
+                    <div class="card_buttons">
+                        <a class="link" href='./pages/productListing.html'><button class="btn btn-primary">See Products</button> </a>
+                    </div>
                 </div>
             </div>
-        </div>
-        
-        <div class="card m-2 up-curve-border">
-            <img class="card-img up-curve-border" src="./assets/chelsea.jpeg"></img>
-            <div class="card_main pl-3 mt-2 mb-1">
-                <div class="card_topic mb-1">Chelsea</div>
-                <div class="card_author">from ₹2399/- only </div>
-            </div>
-            <div class="card_content pl-3 pr-1 mt-2">Men's Chelsea Boots Handcrafted Italian Leather Shoes for Men</div>
-            <div class="card_footer mt-2 mb-2 pl-3">
-                <div class="card_buttons">
-                    <div class="card_button p-1"><i class="fa fa-shopping-cart mr-1"></i> Add to cart</div>
-                    <div class="card_button p-1">Buy Now</div>
+
+            <div class="card m-2 up-curve-border">
+                <img class="up-curve-border" src='./assets/loafers.jpeg' />
```

Comment:
Please use alt attribute as it provides a text description to image and is accessed by the screen readers.